### PR TITLE
Project/lsst verify/bechtol

### PR DESCRIPTION
**Pull request by bechtol**
Notebook to demo the `lsst.verify` Python package. Shows how to create metrics and associated specifications, make measurements, and visualize a summary report.

### CODE REVIEW

**Comment by bechtol** [comment]
I have now verified that the notebook runs on v17.0 and have added some more explanatory notes. I think this is now ready for a content review.

File: Validation/verify_demo.ipynb [hunk 1]
```diff
@@ -0,0 +1,401 @@
...
+    "### Learning Objectives\n",
+    "\n",
+    "After working through and studying this notebook you should be able to\n",
+    "   1. Create custom metrics for your science cases and associate multiple specifications with those metrics\n",
+    "   2. Evaluate metrics and store the output for subsequent analysis\n",
+    "   3. Create customized summary displays for the performance on those metrics relative to your specifications\n",
+    "\n",
+    "### Logistics\n",
+    "\n",
+    "This notebook is intended to be runnable on `lsst-lsp-stable.ncsa.illinois.edu` from a local git clone of https://github.com/LSSTScienceCollaborations/StackClub.\""
```
Comment (SimonKrughoff) [inline Validation/verify_demo.ipynb]: There is an extra double quote at the end of this line.

File: Validation/verify_demo.ipynb [hunk 2]
```diff
@@ -0,0 +1,401 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# What version of the Stack am I using?\n",
+    "! echo $HOSTNAME\n",
+    "! eups list -s | grep lsst_distrib"
```
Comment (SimonKrughoff) [inline Validation/verify_demo.ipynb]: You can just say `eups list -s lsst_distrib`.  You need not grep for the package.

File: Validation/verify_demo.ipynb [hunk 3]
```diff
@@ -0,0 +1,401 @@
...
+    "Create an LSST verify job and add the measurements."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "job = lsst.verify.Job.load_metrics_package()\n",
```
Comment (SimonKrughoff) [inline Validation/verify_demo.ipynb]: Out of curiosity, why not construct from the package name as you did for specs and metrics above?

File: Validation/verify_demo.ipynb [hunk 4]
```diff
@@ -0,0 +1,401 @@
...
+    "job.report(spec_tags=['minimum']).show()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#job.report(metric_tags=['demo']).show()\n",
```
Comment (SimonKrughoff) [inline Validation/verify_demo.ipynb]: I'd like to see the commented out code removed here and below.  The fact that metric_tags don't work is worrying to me.  We should probably file a ticket about it.

Comment (bechtol) [reply Validation/verify_demo.ipynb]: Will do. Here is the submitted ticket for reference: https://jira.lsstcorp.org/browse/DM-18429

File: Validation/verify_demo.ipynb (hunk 3 above)
Comment (bechtol) [reply Validation/verify_demo.ipynb]: Thanks for suggestion. Replacing with the following more compact syntax.
`job = lsst.verify.Job(metrics=metrics, specs=specs)`

File: Validation/verify_demo.ipynb (hunk 2 above)
Comment (bechtol) [reply Validation/verify_demo.ipynb]: Thanks. Done.

File: Validation/verify_demo.ipynb (hunk 1 above)
Comment (bechtol) [reply Validation/verify_demo.ipynb]: Done

File: Validation/verify_demo/metrics/demo_astrometry.yaml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+AstrometricRMS:
+  unit: mas
+  description:
+    Astrometric residual RMS.
+  reference:
+    url: https://example.com/AstroRMS
+  tags:
+    - astrometry
+    - demo
```
[the remark this answers is not in the document]
Comment (bechtol) [reply Validation/verify_demo/metrics/demo_astrometry.yaml]: Done

File: Validation/verify_demo/specs/demo_astrometry/specs.yaml [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+  - "minimum"
+---
+name: "design"
+metric: "AstrometricRMS"
+threshold:
+  operator: "<="
+  unit: "mas"
+  value: 10.0
+tags:
+  - "design"
```
[the remark this answers is not in the document]
Comment (bechtol) [reply Validation/verify_demo/specs/demo_astrometry/specs.yaml]: Done

File: Validation/verify_demo/specs/demo_photometry/specs.yaml [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+---
+name: "design"
+metric: "ZeropointRMS"
+threshold:
+  operator: "<="
+  unit: "mmag"
+  value: 10.0
+tags:
+  - "design"
+
```
[the remark this answers is not in the document]
Comment (bechtol) [reply Validation/verify_demo/specs/demo_photometry/specs.yaml]: Done

**Pull request merged**